homeassistant/components/sensor/wunderground.py

```diff
@@ -0,0 +1,141 @@
+"""Support for Wunderground weather service.
```

Comment:
When I had the closing `"""`, it gave an error `D400: First line should end with a period (not 'd')` so I removed them. Will add them again.